hello.ahk

```diff
@@ -15,103 +15,57 @@ Global VarL := ""
 DetectHiddenWindows, On ; BEcause most AHK script have their window hidden
 SetTitleMatchMode, 2
 
-GroupAdd, SuperMemo, ahk_class TBrowser ;Browser
-GroupAdd, SuperMemo, ahk_class TContents ;Content Window (Knowledge Tree)
-GroupAdd, SuperMemo, ahk_class TElWind ;Element window
-GroupAdd, SuperMemo, ahk_class TElDATAWind ;Element Data window
-GroupAdd, SuperMemo, ahk_class TSMMain ;Toolbar
-
-; these are the lines 1-4 on the menu
-    Menu, MyMenu, Add, L Window(L), MenuHandler  ;MenuHandler is a label -
-    Menu, MyMenu, Add, F SM18, MenuHandler
-    Menu, MyMenu, Add, G Discord, MenuHandler
-    Menu, MyMenu, Add, Run Chrome, MenuHandler
-
-    Menu, MyMenuOne, Add, Generic AI jailbreak, MenuHandlerOne
-    Menu, MyMenuOne, Add, Religious Debater, MenuHandlerOne
-    Menu, MyMenuOne, Add, Catholic Phd, MenuHandlerOne  ;MenuHandler is a label -
-    Menu, MyMenuOne, Add, Atheist Phd, MenuHandlerOne
-    Menu, MyMenuOne, Add, Takeaways from text, MenuHandlerOne
+function_initial_setup()
 
 ; RUN THESE PROGRAMS ON STARTUP
-  if (!WinExist("KeyLog (3).ahk")) {
```

Comment:
I recommend wrapping these identical codes to function so that you will have less lines of code. Example:
	
function_run("KeyLog (3)")
function_run("NiftyWindows")

you can get the idea by looking at what I have changed